### ISSUE TITLE
Protondrive: Invalid signature caused by openpgp: signature made by unknown entity

### ISSUE THREAD

**Issue by tcrasset** [rclone/rclone#7344]
#### What is the problem you are having with rclone?

First of all, thanks to the rclone developers and @henrybear327 for the much awaited proton integration.

I'm having issues downloading some files from my drive. Some files it works, some files it doesn't and show me a 

`Invalid signature cause by openpgp: signature made by unknown entity` error.

I looked through https://github.com/rclone/rclone/issues/6072 as there was a similar issue going around, but it was hard to follow the whole discussion. At the time, there was a config option	`skipSignatureVerification` that seemed to have been removed.

I figured creating a new issue specific to that problem might help centralize the discussion rather than cluttering up the one mentioned above.

FYI, the file I'm trying to download is dated January 12, 2022. It seemed from the discussion that older files were more likely to have signature issues.

Appreciate any help, feel free to ask for more information, I can also lend a hand too @henrybear327, not familiar with Go though (that can change).

#### What is your rclone version (output from `rclone version`)

```text
❯ rclone version
rclone v1.65.0-beta.7392.b296f3780
- os/version: ubuntu 20.04 (64 bit)
- os/kernel: 5.15.0-84-generic (x86_64)
- os/type: linux
- os/arch: amd64
- go/version: go1.21.1
- go/linking: static
- go/tags: none
- ```

#### Which cloud storage system are you using? (e.g. Google Drive)

Protondrive


#### The command you were trying to run (e.g. `rclone copy /tmp remote:tmp`)


`rclone copy protondrive:Photo-profil.jpg .`

#### A log from the command with the `-vv` flag (e.g. output from `rclone -vv copy /tmp remote:tmp`)



Here the full log for a specific file that failed


```text
<7>DEBUG : rclone: Version "v1.65.0-beta.7392.b296f3780" starting with parameters ["rclone" "copy" "protondrive:Photo-profil.jpg" "." "-vv"]
<7>DEBUG : rclone: systemd logging support activated
<7>DEBUG : Creating backend with remote "protondrive:Photo-profil.jpg"
<7>DEBUG : Using config file from "/home/tom/.config/rclone/rclone.conf"
<7>DEBUG : proton drive root link ID 'Photo-profil.jpg': Has cached credentials
<7>DEBUG : proton drive root link ID 'Photo-profil.jpg': Used cached credential to initialize the ProtonDrive API
<7>DEBUG : fs cache: adding new entry for parent of "protondrive:Photo-profil.jpg", "protondrive:"
<7>DEBUG : Creating backend with remote "."
<7>DEBUG : fs cache: renaming cache item "." to be canonical "/home/tom/Documents/personal/rclone-proton-drive-test"
<7>DEBUG : Photo-profil.jpg: Need to transfer - File not found at Destination
<3>ERROR : Photo-profil.jpg: Failed to copy: failed to open source object: Signature Verification Error: Invalid signature caused by openpgp: signature made by unknown entity
<3>ERROR : Attempt 1/3 failed with 1 errors and: failed to open source object: Signature Verification Error: Invalid signature caused by openpgp: signature made by unknown entity
<7>DEBUG : Photo-profil.jpg: Need to transfer - File not found at Destination
<3>ERROR : Photo-profil.jpg: Failed to copy: failed to open source object: Signature Verification Error: Invalid signature caused by openpgp: signature made by unknown entity
<3>ERROR : Attempt 2/3 failed with 1 errors and: failed to open source object: Signature Verification Error: Invalid signature caused by openpgp: signature made by unknown entity
<7>DEBUG : Photo-profil.jpg: Need to transfer - File not found at Destination
<3>ERROR : Photo-profil.jpg: Failed to copy: failed to open source object: Signature Verification Error: Invalid signature caused by openpgp: signature made by unknown entity
<3>ERROR : Attempt 3/3 failed with 1 errors and: failed to open source object: Signature Verification Error: Invalid signature caused by openpgp: signature made by unknown entity
<6>INFO  : 
Transferred:   	         0 B / 0 B, -, 0 B/s, ETA -
Errors:                 1 (retrying may help)
Elapsed time:         8.4s

<7>DEBUG : 7 go routines active
Failed to copy: failed to open source object: Signature Verification Error: Invalid signature caused by openpgp: signature made by unknown entity
```

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by henrybear327** [comment]
Hello @tcrasset, 

Thanks for reporting! 

Yes, indeed, I removed the special config flag since that's just a temporary hack. 

I will make a custom build for you so I can see what part of the verification failed (I think I actually need to make this a debugging feature so it would be easier to see exactly what field is failing, e.g. name, node, etc..)

**Comment by bluboyo** [comment]
Heres another error connected to opengpg

Failed to copy: gopenpgp: error in decrypting: openp
gp: invalid argument: cannot decrypt encrypted session key for key id

**Comment by johanbarelds** [comment]
Did anyone ever found an solution for this?
I experience the same error.
Thanks!
